static/app/components/modals/createOwnershipRuleModal.tsx

```diff
@@ -19,9 +19,7 @@ const CreateOwnershipRuleModal = ({Body, Header, closeModal, ...props}: Props) =
 
   return (
     <React.Fragment>
-      <Header closeButton onHide={closeModal}>
-        {t('Create Ownership Rule')}
-      </Header>
+      <Header closeButton>{t('Create Ownership Rule')}</Header>
```

Comment:
Should `closeButton` be default?

Reply:
Yeah I thought so too, let's change this outside of this